### PR TITLE
Recon all

### PR DESCRIPTION
**Pull request by natacha-beck**

### CODE REVIEW

File: cbrain_task/recon_all/bourreau/recon_all.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@
 # Original author: Natacha Beck
 class CbrainTask::ReconAll < ClusterTask
 
-  Revision_info=CbrainFileRevision[__FILE__] #:nodoc:
```
Comment (prioux) [inline cbrain_task/recon_all/bourreau/recon_all.rb]: typo, there is a G here

File: cbrain_task/recon_all/views/_task_params.html.erb [hunk 2]
```diff
@@ -66,7 +66,11 @@ Workflow directives:<br/>
     options << [ "Process steps  6-23, autorecon2-noaseg", "-autorecon2-noaseg"]
     options << [ "Process steps 12-23, autorecon2-cp",   "-autorecon2-cp" ]
     options << [ "Process steps 15-23, autorecon2-wm",   "-autorecon2-wm" ]
-    options << [ "Process steps 21-23, autorecon2-pial", "-autorecon2-pial" ]
+    if @tool_config.present? && @tool_config.is_version("5.1.0")
+      options << [ "Process steps 21-23, autorecon2-pial", "-autorecon2-pial" ]
+    else
+      options << [ "Process steps 21-23, autorecon2-pial", "-autorecon-pial" ]
```
Comment (prioux) [inline cbrain_task/recon_all/views/_task_params.html.erb]: the description still has  '2' in the option name

Comment (prioux) [inline cbrain_task/recon_all/views/_task_params.html.erb]: also maybe the options for the other stages have also chnaged?

File: cbrain_task/recon_all/bourreau/recon_all.rb [hunk 3]
```diff
@@ -25,7 +25,7 @@
...
+  Revision_info=CbrainFileRevision[__FILE__]G #:nodoc:
```
Comment (prioux) [inline cbrain_task/recon_all/bourreau/recon_all.rb]: the typo is actually here

**Comment by natacha-beck** [comment]
I do both fixes. For other options, I took a look to each of them and apparently it is the only one that have changed...

**Pull request merged**